tests/CMakeLists.txt

```diff
@@ -49,7 +58,11 @@ install(
 set(TEST_UTIL_LIB ${PROJECT_NAME}-test_util)
 add_library(${TEST_UTIL_LIB} STATIC util.cpp)
 target_link_libraries(${TEST_UTIL_LIB} PUBLIC ${PROJECT_NAME})
-target_link_libraries(${TEST_UTIL_LIB} PUBLIC EigenRand::EigenRand)
+if(EigenRand_INCLUDE_DIR)
+  target_include_directories(${TEST_UTIL_LIB} PRIVATE SYSTEM ${EigenRand_INCLUDE_DIR})
+else()
+  target_link_libraries(${TEST_UTIL_LIB} PUBLIC EigenRand::EigenRand)
+endif()
```

Comment:
You can maybe use MODERNIZE_TARGET_LINK_LIBRARIES

Reply:
Just target_link_libraries() should be enough no ? On CMake versions above the minimum

Reply:
`find_path` does not provide a target, just a path, so yes, it would work with MODERNIZE_TARGET_LINK_LIBRARIES